xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -2528,7 +2528,7 @@ void CVideoPlayer::HandleMessages()
 
       double time = msg.GetTime();
       if (msg.GetRelative())
-        time = GetTime() + time;
```

Comment:
For this change, see the discussion here:
https://github.com/peak3d/inputstream.mpd/pull/36#issuecomment-269522413